etc/HIProdOfflineConfiguration.py

```diff
@@ -435,6 +420,29 @@
                  dataset_lifetime=3*30*24*3600,#lifetime for container rules. Default 3 months
                  versionOverride=expressVersionOverride)
 
+addExpressConfig(tier0Config, "HIExpressRawPrime",
+                 scenario=hiRawPrimeScenario,
+                 diskNode="T2_CH_CERN",
+                 data_tiers=["FEVT"],
+                 write_dqm=True,
```

Comment:
```suggestion
                 write_dqm=True,
                 alca_producers=["SiStripCalMinBias", "SiStripCalMinBiasAAG"],
```

as per https://github.com/dmwm/T0/pull/4875#discussion_r1331866459 ?

Reply:
> as per https://github.com/dmwm/T0/pull/4875#discussion_r1331866459 ?

Yup. +1 @germanfgv